genie-web/src/main/resources/db/migration/h2/V4_0_0__Genie_4.sql

```diff
@@ -155,3 +155,16 @@ ALTER INDEX `JOB_APPLICATIONS_REQUESTED_APPLICATION_ID_INDEX`
   RENAME TO `JOB_REQUESTED_APPLICATIONS_APPLICATION_ID_INDEX`;
 ALTER INDEX `JOB_APPLICATIONS_REQUESTED_JOB_ID_INDEX`
   RENAME TO `JOB_REQUESTED_APPLICATIONS_JOB_ID_INDEX`;
+
+CREATE TABLE `agent_connections` (
+  `id`              BIGINT(20)    NOT NULL     AUTO_INCREMENT,
+  `created`         DATETIME(3)   NOT NULL     DEFAULT CURRENT_TIMESTAMP(3),
+  `updated`         DATETIME(3)   NOT NULL     DEFAULT CURRENT_TIMESTAMP(3),
+  `entity_version`  INT(11)       NOT NULL     DEFAULT '0',
+  `job_id`          VARCHAR(255)  NOT NULL,
```

Comment:
Looks like the other conversation got replaced by changes. I think you'd said something about not wanting to do the foreign key cause then you require it to be there. That's not true cause you could do a foreign key on this field but still make this field optional. It just ensures the referential integrity of the value stored here (e.g. the job this connection references actually exists). It's not a huge deal and we could change it later or whatever if we decide to even keep this here.